### DISCUSSION TITLE
Activate Logging for Maputnik Server

### DISCUSSION THREAD

**Discussion by jlpoolen** [maputnik/editor#907] [Q&A]
I'm running my own Maputnik server and would like to know how to activate logging and where the log files are.  In the last hour, I've been trying some tests and I could not find any logs in my build directory, /usr/local/src/maputnik, or on my Debian server's log area: /var/log.

How may I activate logging?

**Comment by HarelM** [comment]
I don't know what maputnik server is.
It mainly uses the browser. 
I would recommend using the online version instead of something local.

**Reply by jlpoolen** [reply]
> I don't know what maputnik server is. It mainly uses the browser. I would recommend using the online version instead of something local.

The server is some sort of Node package that listens on port 8888.  The server is started by running this command in the project's root directory.  (It is unclear whether you have to be root or not, I find it works running as a user, i.e. jlpoole.)
`npm run start -- --host 0.0.0.0`

Here's a screenshot of what my console shows after running the command (repeated on the last line since running the command causes the screen to clear to display the information above).
![2024-06-19_06-42](https://github.com/maplibre/maputnik/assets/227313/0c6254b3-3f54-47aa-87ef-c6d3a9d7e1a3)

**Reply by HarelM** [reply]
It is a static files server that only servers the html JavaScript and CSS file. There are no log to be found there. Look in the browser console and local storage.
